.github/workflows/trivy.yaml

```diff
@@ -0,0 +1,17 @@
+name: Trivy
+
+on:
+  push:
+    branches:
+      - master
+      - branch/*
+  pull_request:
+  merge_group:
```

Comment:
Is this supposed to run on each pull request and in the merge group? How long does this check take?

Reply:
Looking at the history about 2 minutes, should be ok then.

Reply:
Yes -- the goal is to eventually (once we've gained some comfort with it) make this check blocking merges.
The goal is to catch newly committed misconfigurations before they're merged.